kbase-extension/static/kbase/js/common/sdk.js

```diff
@@ -53,10 +62,6 @@ define(['common/props'], (Props) => {
         })();
     }
 
-    function updateNullValue(converted) {
-        converted.data.nullValue = nullValue(converted);
-    }
```

Comment:
I was getting annoyed with these functions that just redirect to another function, so replaced them with a direct call.